src/dotnet/APIView/APIViewWeb/Managers/ReviewManager.cs

```diff
@@ -330,66 +331,62 @@ public async Task ApproveReviewAsync(ClaimsPrincipal user, string reviewId, stri
-            }
-
-            var url = "https://apiview-gpt.azurewebsites.net/python";
+            var codeLines = codeFile.CodeFile.GetApiLines(skipDocs: true);
+            var url = $"https://apiview-gpt.azurewebsites.net/{LanguageServiceHelpers.GetLanguageAliasForCopilotService(revision.Language)}";
             var client = new HttpClient();
             client.Timeout = TimeSpan.FromMinutes(20);
-            var payload = new
+            var payload = new Dictionary<string, object>
             {
-                content = reviewText.ToString()
+                { "target", String.Join("\\n", codeLines.Select(item => item.lineText.Trim())) },
+                { "base", string.Empty },
+                { "diff", string.Empty }
             };
 
             var result = new AIReviewModel();
             try {
                 var response = await client.PostAsync(url, new StringContent(JsonSerializer.Serialize(payload), Encoding.UTF8, "application/json"));
                 response.EnsureSuccessStatusCode();
                 var responseString = await response.Content.ReadAsStringAsync();
-                var responseSanitized = JsonSerializer.Deserialize<string>(responseString);
-                result = JsonSerializer.Deserialize<AIReviewModel>(responseSanitized);
+                result = JsonSerializer.Deserialize<AIReviewModel>(responseString);
             }
             catch (Exception e ) {
                 throw new Exception($"Copilot Failed: {e.Message}");
             }
            
             // Write back result as comments to APIView
-            foreach (var violation in result.Violations)
+            foreach (var comment in result.Comments)
             {
-                var codeLine = codeLines[violation.LineNo];
-                var comment = new CommentItemModel();
-                comment.CreatedOn = DateTime.UtcNow;
-                comment.ReviewId = reviewId;
-                comment.APIRevisionId = revisionId;
-                comment.ElementId = codeLine.ElementId;
+                var codeLine = codeLines[comment.LineNo - 1];
+                var commentModel = new CommentItemModel();
+                commentModel.CreatedOn = DateTime.UtcNow;
+                commentModel.ReviewId = reviewId;
+                commentModel.APIRevisionId = apiRevisionId;
+                commentModel.ElementId = codeLine.lineId;
                 //comment.SectionClass = sectionClass; // This will be needed for swagger
-
+            
                 var commentText = new StringBuilder();
-                commentText.AppendLine($"Suggestion: `{violation.Suggestion}`");
+                commentText.AppendLine(comment.Comment);
                 commentText.AppendLine();
-                commentText.AppendLine(violation.Comment);
-                foreach (var id in violation.RuleIds)
+                commentText.AppendLine();
+                commentText.AppendLine($"Suggestion : {comment.Suggestion}");
```

Comment:
```suggestion
                commentText.AppendLine($"Suggestion : `{comment.Suggestion}`");
```

Generally, the suggestion should be wrapped in backticks (it's supposed to be code).

Reply:
Also, `comment.Suggestion` could be null, and it should only render this at all if it non-null and non-empty string.